_includes/sqlstyle.guide.md

```diff
@@ -270,12 +273,41 @@ SELECT r.last_name,
           FROM champions AS c
          WHERE c.last_name = r.last_name
            AND c.confirmed = 'Y') AS last_championship_year
-  FROM riders AS r
- WHERE r.last_name IN
-       (SELECT c.last_name
-          FROM champions AS c
-         WHERE YEAR(championship_date) > '2008'
-           AND c.confirmed = 'Y');
+FROM riders AS r
+WHERE r.last_name IN
+      (SELECT c.last_name
+         FROM champions AS c
+        WHERE YEAR(championship_date) > '2008'
+          AND c.confirmed = 'Y');
```

Comment:
my pref here is for subqueries (which i rarely use in favor of WITHs, and perhaps we should call that out) is to have the paren on a new line, just like WITHs